_gsocprojects/2020/project_MCnet.md

```diff
@@ -0,0 +1,14 @@
+---
+project: MCnet
+layout: default
+logo: mcnet-logo.png
+description: |
+  [MCnet](http://montecarlonet.org) is a collaboration of the major collider event
+  modelling tools used at the [Large Hadron Collider](http://home.web.cern.ch/topics/large-hadron-collider) (LHC)
+  and beyond. These so-called "MC event generators" are theoretical physics tools that
```

Comment:
"These" implies that you mentioned somewhere event generators. How about: "The collaboration is focused on developing so-called "MC event generators", which are theoretical physics tools ...